spec/hermod/xml_section_builder/datetime_node_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "minitest_helper"
+
+module Hermod
+  describe XmlSection do
+
+    DateTimeXml = XmlSection.build(formats: {datetime: "%Y-%m-%d %H:%M:%S"}) do |builder|
```

Comment:
Should this pick up the format from the default formats? (I can't remember enough of hermod to figure this out sorry)